bounties/level-2/sep10-javascript-client.md

```diff
@@ -0,0 +1,63 @@
+# JavaScript SEP-0010 Client Reference Implementation
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 800 XLM | 800 XLM | Capped (3)
```

Comment:
I think it should be more for both parties. The result of this bounty could end up being used by many readers of the SEP so it should be solid and being put up with the right effort. There are many conditions in the SEP that need to be a) implemented and b) thoroughly reviewed.

Reply:
Agreed. Still not sure what a good award amount would be, though.